src/key-derivation.ts

```diff
@@ -2,6 +2,7 @@ import BN from "bn.js";
 import {getHkdf} from "./crypto";
 // @ts-ignore
 import {SHA256} from "bcrypto";
+import {Buffer} from "buffer";
```

Comment:
this is to ensure whoever is using your library don't need Buffer polyfill. It will make library work out of box. Common practice apparently (take a look at libp2p :) )

Reply:
:+1: